### DISCUSSION TITLE
Qiita の治安について

### DISCUSSION THREAD

**Discussion by ninchu-toshu** [increments/qiita-discussions#377] [要望・アイデア・不具合]
通報が機能しているようには見えません。

明らかに誰が見ても社会的に不適切な暴言とわかるコメントが多く残っていますが、通報フォームから通報しても、問い合わせしても放置されたままです。

人が足りていないのでしょうか？

**Comment by tanaka-asami415** [comment]
@ninchu-toshu 
コミュニティガイドラインに沿って対応はしており、運営が強制的に関与するかどうかは、
1ユーザーさまからの通報だけでなく、複数アカウントからの通報等も合わせての総合判断となります。

また、【初めにお読みください】 #116 にも記載しておりますが
Discussions上では利用規約やコミュニティガイドライン違反の報告については受付しておりません。

皆様から通報いただいた内容は順次対応しております。
ご理解の程よろしくお願い申し上げます。
